Wirtualnik.Frontend/components/common/PopupMenu.vue

```diff
@@ -96,58 +100,74 @@ export default class PopupMenu extends Vue {
         return event;
     }
 
+    logout() {
+        this.$store.commit(`auth/${TokenMutations.SET_TOKEN}`, '');
+    }
 
-    logout(){
-      this.$store.commit(`auth/${TokenMutations.SET_TOKEN}`, '')
-
+    login() {
+        var token = '';
```

Comment:
Use `let` everywhere when value can be changed, `const` otherwise. Use `var` only when required.